benchto-service/src/main/resources/db/migration/V006__metrics.sql

```diff
@@ -0,0 +1,111 @@
+CREATE TABLE metrics
+(
+    id   BIGSERIAL PRIMARY KEY NOT NULL,
+    name VARCHAR(64)           NOT NULL,
+    unit VARCHAR(16)           NOT NULL
+);
+
+CREATE TABLE metric_attributes
+(
+    metric_id BIGINT       NOT NULL,
+    name      VARCHAR(255) NOT NULL,
```

Comment:
```suggestion
    name      VARCHAR(64) NOT NULL,
```

Reply:
Why?

Reply:
It seems the attributes names are quite short, and all the other entities limit their names to 64 characters.

Reply:
But 255 is consistent with all other attribute tables.